WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -363,9 +361,6 @@ extension MainTabBarController {
 
     /// Presents the order details if the `note` is for an order push notification.
     ///
-    /// For Product Review notifications, that is now handled by `ReviewsCoordinator`. This method
-    /// should also be moved to a similar `Coordinator` in the future too.
```

Comment:
super nit: might be worth mentioning where we can find the code for product review notifications like before 🙂